packages/admin/src/routing/types.ts

```diff
@@ -1,20 +1,29 @@
+import { RoutingParameter } from './RoutingParameter'
+
 export interface SelectedDimension {
 	[key: string]: string[]
 }
 
-export interface PageParameters {
-	[key: string]: string | PageParameters | undefined
+export type RequestParameterValue<Extra extends RoutingParameter = never> = string | undefined | Extra;
```

Comment:
Why is it generic? Wouldn't it be better (more readable) to have the extra union on "use-side"?